.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_ROOT_PASSWORD=root_password
+MYSQLDB_USER=user_name
+MYSQLDB_PASSWORD=user_password
+MYSQLDB_DATABASE=db_name
+MYSQLDB_LOCAL_PORT=3309
+MYSQLDB_DOCKER_PORT=3306
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005

```

Comment:
Add this file to `.gitignore`. Putting a file with credentials in a public repository is not secure. Instead, add a file `.env.sample` with a list of variables without secure values.